src/drone/agent/executor.rs

```diff
@@ -195,6 +232,7 @@ impl Executor {
 
                     if state.running() {
                         self.start_log_loop(&spawn_request.backend_id);
+                        self.start_stats_loop(&spawn_request.backend_id)
```

Comment:
nit: add semicolon at the end.

This is one of the few places where Rust doesn't care if there's a semicolon, but I prefer to use semicolons in those cases because a reader can look at a line in isolation and see that the return value is discarded.